lua/shine/extensions/afkkick.lua

```diff
@@ -173,6 +175,21 @@ function Plugin:CheckConnectionAllowed( ID )
 end
 
 --[[
+	Notify other plugins when changing client's AFK status.
+]]
+local function ChangeIsAFK( Client, IsAFK )
+	local DataTable = Shine.Plugins.afkkick.Users[ Client ]
+	if not DataTable then return end
+
+	DataTable.IsAFK = IsAFK
+	if LastAnnouncedAfkStatuses[ Client ] == nil or LastAnnouncedAfkStatuses[ Client ] ~= IsAFK then
```

Comment:
Can this not simply be:
```lua
if DataTable.IsAFK ~= IsAFK then
    DataTable.IsAFK = IsAFK
    Shine.Hook.Call( "AFKChanged", Client, IsAFK )
end
```
rather than having an extra table that has to be managed and that is outside the plugin?

I think this should also be a method, so that the users table can be referenced through `self` rather than traversing the plugins table to get back to it. It could also just use the local `Plugin`, but I don't think that's as clean.